### PR TITLE
Creating a file for Rome, Italy

### LINKED ISSUE #29: Create lmosca hometown file
## Rome, Italy
- [x] Create a branch
- [x] Create a file
- [x] Add a commit
- [x] Create a Pull Request
- [ ] Have a conversation
- [ ] Get a :+1:
- [ ] :ship: it (a.k.a. merge)

### PR DESCRIPTION
**Pull request by lmosca**
This PR resolves #29

### CODE REVIEW

File: lmosca-rome.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+## Rome, Italy
+
+### Places to eat
+
+### Fun things to do
```
Comment (yading2014) [inline lmosca-rome.md:7]: @lmosca Cannot wait to hear more about Rome!!

Comment (la-moviestarplanet) [inline lmosca-rome.md:7]: @lmosca Romeeee! wuhu!

Comment (umanganiello) [inline lmosca-rome.md:7]: @lmosca This line is not correct #littlepete

File: lmosca-rome.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+## Rome, Italy
+
+## CIAO
```
Comment (Simoruty) [inline lmosca-rome.md]: @lmosca sostato io

**Pull request merged**